stingray/lightcurve.py

```diff
@@ -180,7 +203,7 @@ def __neg__(self):
         >>> lc_new.counts
         array([100, 100, 100])
         """
-        lc_new = Lightcurve(self.time, -1*self.counts)
+        lc_new = Lightcurve(self.time, -1*self.counts, gti=self.gti)
```

Comment:
is the `gti=self.gti` not redundant here? I mean, the GTIs don't change, and there's no possibility to put in new GTIs.